library/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.devbrackets.android.exomedia">
+<manifest package="com.devbrackets.android.exomedia"
+          xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-sdk/>
+
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
The library itself doesn't handle foreground services, that's left up to PlaylistCore. As such this should be in the demo apps manifest instead.